dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -1451,7 +1530,7 @@ void ScrollViewer::UpdateScrollControllersVisibility(
     }
 }
 
-bool ScrollViewer::IsLoaded()
+bool ScrollViewer::IsLoaded() const
 {
     return winrt::VisualTreeHelper::GetParent(*this) != nullptr;
```

Comment:
>return winrt::VisualTreeHelper::GetParent(*this) != nullptr; [](start = 4, length = 60)

There is an isLoaded property that was added in RS5 (i think) we should use that here instead of visual tree helper.

Reply:
Yep - I moved some existing code into a new static bool SharedHelpers::IsFrameworkElementLoaded(winrt::FrameworkElement const& frameworkElement).